src/macros.rs

```diff
@@ -32,7 +32,7 @@ macro_rules! load_yaml {
 }
 
 /// Convenience macro getting a typed value `T` where `T` implements [`std::str::FromStr`] from an
-/// argument value. This macro returns a `Result<T,String>` which allows you as the developer to
+/// argument value. This macro returns a [`clap::Result<T>`] which allows you as the developer to
 /// decide what you'd like to do on a failed parse. There are two types of errors, parse failures
 /// and those where the argument wasn't present (such as a non-required argument). You can use
 /// it to get a single value, or a iterator as with the [`ArgMatches::values_of`]
```

Comment:
While you are on this, can you please add a full-stop to the end of sentence?

Reply:
I thought it was intentional - added in the newest commit